serverjs/render.js

```diff
@@ -82,6 +82,16 @@ const render = (req, res, page, reactProps = {}, options = {}) => {
 
   reactProps.loginCallback = req.baseUrl + req.path;
 
+  if (!options.metadata) {
+    options.metadata = [];
+  }
+  if (options.metadata.filter((data) => data.property === 'og:image').length === 0) {
+    options.metadata.push({
+      property: 'og:image',
+      content: '/content/sticker.png',
+    });
+  }
+
   return res.render('main', {
```

Comment:
Replacing a falsy `options.metadata` with `null` is redundant, because, since the template does a truthiness check anyway.

Reply:
(the actual check is actually below the snippet, because it's too far from the changes, but I figured it was worth mentioning)